flutter-studio/flutter-studio.iml

```diff
@@ -48,5 +48,6 @@
     <orderEntry type="library" name="com.android.tools:common" level="project" />
     <orderEntry type="library" name="com.android.tools:repository" level="project" />
     <orderEntry type="library" name="studio-analytics-proto" level="project" />
+    <orderEntry type="module" module-name="intellij.groovy.psi" />
```

Comment:
Is this used? I don't see any references in the code to a groovy package.

Reply:
Yes. I tried taking it out and got a compile error.